.idea/misc.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="FrameworkDetectionExcludesConfiguration">
+    <file type="web" url="file://$PROJECT_DIR$" />
+  </component>
   <component name="IdProvider" IDEtalkID="9BC0DE4049720C1246C2D3B835C6F7C3" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" project-jdk-name="19" project-jdk-type="JavaSDK">
```

Comment:
I think this should be `project-jdk-name="17"`. When I check out this PR, IntelliJ also changed that automatically.

Reply:
Each user probably would have to configure a JDK for the project.

Reply:
Yes I agree. But since the language level is JDK-17 and this file is part of the version control, I would recommend we synchronize the `project-jdk-name` with the `languageLevel`, in this case `17`.

Reply:
I've changed it to "17".